tools/dynamicdatamonitor/DynamicDataMonitor.mpc

```diff
@@ -1,6 +1,7 @@
-project(Dynamic_Data_Monitor): dcps_rtps_udp {
-  requires += no_opendds_safety_profile
+project(Dynamic_Data_Monitor): dcps_rtps_udp, install {
   exename = dynamic-data-monitor
+  exeout = $(DDS_ROOT)/bin
```

Comment:
```suggestion
  exename = dynamic_data_monitor
  exeout = $(DDS_ROOT)/bin
```
Nothing else uses dashes

Reply:
This was [deliberate change I had Clayton make](https://github.com/objectcomputing/OpenDDS/pull/3125#discussion_r748770003).

I don't think we should use dashes (or capital letters) in new command names because it's unusual. For example on my system: `ls -1 /usr/bin | grep '_' | wc -l` gives 146, with the large majority of them either having `x86_64` in their name or are `dh_*` commands used in building debian packages. `ls -1 /usr/bin | grep '-' | wc -l` gives 702.

Also assuming standard PC-based keyboard, one less button to press when typing the name.

Reply:
It's not a change I'd like us to take on right now.

Reply:
I don't think there's any negative with it when the existing commands are not consistent (underscores vs camel case vs nothing). Also unlike most of the other commands this is a command that users are more like to be directly invoked on the command like so I think it's even more critical to get the name right when we're introducing it. `opendds_idl` and `dcpinfo_dump` are the only commands that have underscores. `opendds_idl` isn't normally invoked manually and `dcpsinfo_dump` is a minor utility as far as I'm aware.

Reply:
I think we're looking at different data here.  As far as I can tell, executables that are installed in bin and use a separator use underscore.